### DISCUSSION TITLE
resizing nodes on the ui

### DISCUSSION THREAD

**Discussion by glacier434** [Comfy-Org/ComfyUI#2478] [Ideas]
can you make it so that I can resize from bottom left, top left, top right? 
Currently it is only bottom right. 
TY

**Comment by IProduceWidgets** [comment]
Id also like to request a single-axis mode from the edges.

**Comment by ltdrdata** [comment]
Please move this feature request to here.
https://github.com/Comfy-Org/ComfyUI_frontend
